crates/bitwarden-exporters/src/cxf/export.rs

```diff
@@ -135,11 +135,198 @@ fn convert_totp(totp: Totp) -> TotpCredential {
+        );
+
+        // Create PersonName credential only if name-related fields are present
+        if has_name_fields {
+            let person_name = PersonNameCredential {
+                title: identity.title.clone().map(|v| v.into()),
+                given: identity.first_name.clone().map(|v| v.into()),
+                given_informal: None,
+                given2: identity.middle_name.clone().map(|v| v.into()),
+                surname_prefix: None,
+                surname: identity.last_name.clone().map(|v| v.into()),
+                surname2: None,
+                credentials: identity.company.clone().map(|v| v.into()),
+                generation: None,
+            };
+
+            credentials.push(Credential::PersonName(Box::new(person_name)));
+        }
+
+        // Create Address credential only if address fields are present
+        if has_address_fields {
+            // Combine address lines with newlines as per CXF spec
+            let street_address = {
+                let address_lines: Vec<String> = [
+                    identity.address1.as_ref(),
+                    identity.address2.as_ref(),
+                    identity.address3.as_ref(),
+                ]
+                .into_iter()
+                .flatten()
+                .cloned()
+                .collect();
+
+                if address_lines.is_empty() {
+                    None
+                } else {
+                    Some(address_lines.join("\n"))
+                }
+            };
+
+            let address = AddressCredential {
+                street_address: street_address.map(|v| v.into()),
+                city: identity.city.clone().map(|v| v.into()),
+                territory: identity.state.clone().map(|v| v.into()),
+                country: identity.country.clone().map(|v| v.into()),
+                tel: identity.phone.clone().map(|v| v.into()),
+                postal_code: identity.postal_code.clone().map(|v| v.into()),
+            };
+
+            credentials.push(Credential::Address(Box::new(address)));
+        }
+
+        // Create document credentials based on available fields - each type separately (to preserve
+        // all data)
+
+        // Create Passport credential if passport number is present
+        if let Some(passport_number) = identity.passport_number {
+            let passport = PassportCredential {
+                issuing_country: identity.country.clone().map(|v| v.into()),
+                nationality: None,
+                full_name: full_name.clone().map(|v| v.into()),
+                birth_date: None,
+                birth_place: None,
+                sex: None,
+                issue_date: None,
+                expiry_date: None,
+                issuing_authority: None,
+                passport_type: None,
+                passport_number: Some(passport_number.into()),
+                national_identification_number: identity.ssn.clone().map(|v| v.into()),
+            };
```

Comment:
@Hinton What do we want to do with the values like `birth_date`? During import we map to custom fields.

Do we want to see if there is a matching custom field or do we just want to map all custom fields to a customfields credential?

Reply:
I think in this stage we can just map it to customfields.